inc/customizer/layouts.php

```diff
@@ -98,6 +98,7 @@ public function __construct() {
 
 		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts' ) );
 		add_action( 'customize_register',    array( $this, 'customize_register' ) );
+		add_action( 'add_post_meta',         array( $this, 'page_builder_layout' ), 10, 3 );
```

Comment:
@fjarrett You actually only need 2 args since you are not using `$meta_value`, but I really really don't mind that much.